README.md

```diff
@@ -1,22 +1,18 @@
 # WhateverRed
 
-AMD iGPU support [Lilu](https://github.com/acidanthera/Lilu) plug-in.
+An AMD iGPU support plugin for Lilu.
 
-The Source Code of this Original Work is licensed under the `Thou Shalt Not Profit License version 1.0`. See `LICENSE`
+This original work's source code is licensed under the `Thou Shalt Not Profit License version 1.0`. See `LICENSE`
 
 ## ⚠️ IMPORTANT ⚠️
 
-**This Kernel Extension contains remote debug logging functionality via our custom "NetDbg" software; you must specify a valid NetDbg server IP or explicitly disable the functionality as described below.**
+This kernel extension includes remote debugging logging functionality through our custom "NetDbg" software. You must specify a valid NetDbg server IP or disable the functionality as described below.
```

Comment:
includes -> contains
debugging logging is bad grammar. Change it to "debug log"

Reply:
```suggestion
This kernel extension currently contains remote debug log functionality through our custom "NetDbg" software. You must specify a valid NetDbg server IP or disable the functionality as described below.
```